stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/internal/module/ConnectionModule.kt

```diff
@@ -288,12 +289,57 @@ internal class CoordinatorAuthInterceptor(
     }
 }
 
-internal class HeadersInterceptor : Interceptor {
+internal class HeadersInterceptor(
+    context: Context,
+) : Interceptor {
+    private val userAgent by lazy { buildUserAgent(context) }
+
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
             .newBuilder()
+            .addHeader("User-Agent", userAgent)
             .addHeader("X-Stream-Client", StreamVideo.buildSdkTrackingHeaders())
```

Comment:
The `x-stream-client` header contains similar info, maybe we can add the extra data in it?

Reply:
On chat and iOS its the same so its fine